Examples/Scripts/Python/gaussian_sum_filter.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python3
+from pathlib import Path
+from typing import Optional, Union
+
+from acts.examples import Sequencer, GenericDetector, RootParticleReader
+
+import acts
+
+from acts import UnitConstants as u
+
+
+def runCKFTracks(
+    trackingGeometry,
+    decorators,
+    geometrySelection: Path,
+    digiConfigFile: Path,
+    field,
+    outputDir: Path,
+    truthSmearedSeeded=False,
+    truthEstimatedSeeded=False,
+    outputCsv=True,
+    inputParticlePath: Optional[Path] = None,
+    s=None,
+):
+
+    from particle_gun import addParticleGun, EtaConfig, PhiConfig, ParticleConfig
+    from fatras import addFatras
+    from digitization import addDigitization
+
+    s = s or acts.examples.Sequencer(
+        events=100, numThreads=-1, logLevel=acts.logging.INFO
+    )
+
+    for d in decorators:
+        s.addContextDecorator(d)
+
+    rnd = acts.examples.RandomNumbers(seed=42)
+    outputDir = Path(outputDir)
+
+    if inputParticlePath is None:
+        s = addParticleGun(
+            s,
+            EtaConfig(-2.0, 2.0),
+            ParticleConfig(4, acts.PdgParticle.eMuon, True),
+            PhiConfig(0.0, 360.0 * u.degree),
+            multiplicity=2,
+            rnd=rnd,
+        )
+    else:
+        acts.logging.getLogger("GSF Example").info(
+            "Reading particles from %s", inputParticlePath.resolve()
+        )
+        assert inputParticlePath.exists()
+        s.addReader(
+            RootParticleReader(
+                level=acts.logging.INFO,
+                filePath=str(inputParticlePath.resolve()),
+                particleCollection="particles_input",
+                orderedEvents=False,
+            )
+        )
+
+    s = addFatras(
+        s,
+        trackingGeometry,
+        field,
+        rnd=rnd,
+    )
+
+    s = addDigitization(
+        s,
+        trackingGeometry,
+        field,
+        digiConfigFile=digiConfigFile,
+        rnd=rnd,
+    )
+
+    selAlg = acts.examples.TruthSeedSelector(
```

Comment:
Could this use `addSeeding` and use its arguments to configure this seeding strategy?